hack/build-and-push.sh

```diff
@@ -50,11 +50,8 @@ for TASK in $SCRIPTDIR/../tasks/*.yaml; do
         rm $TASK_TEMP
     fi
     # Replace appstidio-utils placeholder by newly build appstudio-utils image
-    if yq -M -e e ".spec.steps[0].image==\"appstudio-utils\"" $TASK &>/dev/null; then
-        yq -M e ".spec.steps[0].image=\"$APPSTUDIO_UTILS_IMG\"" $TASK >> $TASK_TEMP
-    else
-        cat $TASK >> $TASK_TEMP
-    fi
+    yq -M e ".spec.steps[] |= select(.image == \"appstudio-utils\").image=\"$APPSTUDIO_UTILS_IMG\"" \
```

Comment:
cc @Michkov

Reply:
I had to squint pretty hard, but I think I get it now.

Reply:
Given the amount of time I spent on this, it's probably one of the most expensive lines I ever wrote.